extensions-core/kinesis-indexing-service/src/test/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisorTest.java

```diff
@@ -3735,7 +3736,7 @@ public void testGetCurrentTotalStats()
 
   @Test
   public void testDoNotKillCompatibleTasks()
-      throws InterruptedException, EntryExistsException
+      throws InterruptedException, EntryExistsException, DruidException
```

Comment:
I am curious as to why you declared DruidException here. It's a runtime exception.